### DISCUSSION TITLE
LSE-3 support

### DISCUSSION THREAD

**Discussion by krzpl** [StephanJoubert/home_assistant_solarman#76] [Ideas]
Is there possibility to choose version of logger in configuration? (LSE-3 and LSW-3)? Using LSE-3 as device and configuration from LSW-3 I cannot connect to logger. solarman_status_connection is still disconnected. Maybe there's some other way to connect

**Comment by adnovea** [comment]
Maybe you can check the [post https://github.com/StephanJoubert/home_assistant_solarman/issues/247#issuecomment-1591817882](https://github.com/StephanJoubert/home_assistant_solarman/issues/247#issuecomment-1590665761)
